python/friesian/src/bigdl/friesian/feature/table.py

```diff
@@ -264,14 +263,11 @@ def random_split(self, weights, seed=None):
 
         :return A list of Tables
```

Comment:
Refer to split and enhance the doc a little bit?

Reply:
In pydoc, `split = random_split` will generate a line of doc like `split=random_split(self, weights, seed=None)`.
Is there still a need for additional doc in this case?

Reply:
I mean merging the doc of random split and split, to make the doc clearer,
For example
```
Randomly split the Table into multiple Tables with the provided weights for train, validation and test.
```